_studio/shared/umc/codec/h265_dec/src/umc_h265_va_packer_vaapi.cpp

```diff
@@ -113,7 +113,7 @@ namespace UMC_HEVC_DECODER
         MFX_INTERNAL_CPY(pipelineBuf, &vpVA->m_pipelineParams, sizeof(VAProcPipelineParameterBuffer));
 
         pipelineBuf->surface = m_va->GetSurfaceID(sliceInfo->m_pFrame->m_index); // should filled in packer
-        pipelineBuf->additional_outputs = (VASurfaceID*)vpVA->GetCurrentOutputSurface();
+        pipelineBuf->input_color_properties.chroma_sample_location = VA_CHROMA_SITING_HORIZONTAL_LEFT | VA_CHROMA_SITING_VERTICAL_CENTER;
```

Comment:
same

Reply:
Upadate code comment: // To keep output aligned, decode downsampling use this fixed combination of chroma sitting type